packages/shortest/src/core/runner/index.ts

```diff
@@ -457,6 +456,10 @@ export class TestRunner {
             await browserTool.getNormalizedComponentStringByCoords(x, y);
 
           if (componentStr !== step.extras.componentStr) {
+            this.log.trace("Component UI elements mismatch", {
```

Comment:
Noticed that the `Log in` test cannot use the cache version, possible related to #285.
Will investigate separately.